core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletPort.scala

```diff
@@ -50,6 +50,22 @@ trait Inlet extends StreamletPort
  */
 trait Outlet extends StreamletPort
 
+/**
+ * A default error handler. This error handler just logs bad message and skips them.
+ *
+ */
+object CodecInlet {
+
+  val logger = LoggerFactory.getLogger(this.getClass)
+
+  def logAndSkip[T](message: Array[Byte], cause: Throwable): Option[T] = {
+    logger.error(s"Input data $message can not be transformed and will be skipped")
```

Comment:
```suggestion
```

Reply:
I just don't think logging bytes like this is useful. Users can do that if they want in their errorHandler.